mssqlcli/main.py

```diff
@@ -1,21 +1,25 @@
 from __future__ import unicode_literals
 from __future__ import print_function
+from builtins import input
+from mssqlcli.config import config_location
+from mssqlcli.__init__ import __version__
+from mssqlcli.mssqlclioptionsparser import create_parser
+
+import mssqlcli.telemetry as telemetry_session
 
 import click
 import getpass
+import gettext
 import os
 import sys
 
 
-from builtins import input
-
-from mssqlcli.config import config_location
-from mssqlcli.__init__ import __version__
-from mssqlcli.mssqlclioptionsparser import create_parser
-import mssqlcli.telemetry as telemetry_session
-
 click.disable_unicode_literals_warning = True
 
+translation = gettext.translation('mssql-cli', localedir='locale', fallback=True)
+translation.install()
+_ = translation.gettext

```

Comment:
You shouldn't need this line - doesn't install take care of that?

Reply:
I don't understand. Did you mean `_` method is automatically defined by `translation.install()` ?

Reply:
Correct. See https://docs.python.org/3/library/gettext.html#gettext.install